src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -186,6 +186,7 @@ Do you want to start the scheduler service now? [Y/n]  y
 If a problem occurs, check output in /var/log/proactive/scheduler
 ....
 
+WARNING: When restarting, ProActive populates the catalog of the new installation with the default objects, by preserving those modified by the user from the old installation (based on the last commit). 
```

Comment:
WARNING: Once the upgrade is performed and the new version started for the first time, the ProActive server will upgrade <<_glossary_catalog,Catalog>> objects to the new version. This procedure ensures that any object modified manually by a user will not be upgraded.